backend/ibutsu_server/scripts/mongo2postgres.py

```diff
@@ -1,425 +0,0 @@
-#!/bin/env python3
```

Comment:
With this gone can we remove the dep on `pymongo`?

Reply:
Whoops, didn't see your comment till now. Yes we can, I'll do that here too.